vulture/core.py

```diff
@@ -461,26 +501,45 @@ def _define(
         confidence=DEFAULT_CONFIDENCE,
         ignore=None,
     ):
+        def ignored(lineno):
+            if (ignore and ignore(self.filename, name)) or _match(
+                name, self.ignore_names
+            ):
+                return True
+
+            # if name not ignored, check if annotated with "# noqa"
+            if typ == "property" and sys.version_info < (3, 8):
+                # for python < 3.8, increment as many line numbers as there are
+                # decorators on the method.
+                # python3.8 onwards, lineno for property is the same as that of
+                # the decorated method.
+                lineno = lineno + len(first_node.decorator_list)
```

Comment:
I think this means that the line numbers we report for unused properties now differ from the line numbers that must have noqa on them. This should be fixed in a separate PR. Can you add a TODO to the TODO file, please?

Reply:
I raised #200 :D -- I would be happy to rebase this branch, once we get it merged.